### PR TITLE
Reduce the size of fiftyone package

### PR DESCRIPTION
**Pull request by minhtuev**
## What changes are proposed in this pull request?

We want to get fiftyone package size to 250MB. 

Remove tutorials and recipes from setup.py can help to do that.

## How is this patch tested? If it is not, please explain why.

Rebuild `fiftyone` and check for size; this should not be a breaking change.

### Is this a user-facing change that should be mentioned in the release notes?

-   [x] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [x] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

## Summary by CodeRabbit

- **Refactor**
	- Streamlined the installation process by excluding certain non-essential packages.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -138,8 +138,7 @@ def get_install_requirements(install_requires, choose_install_requires):
     long_description_content_type="text/markdown",
     packages=find_packages(
         exclude=["app", "eta", "package", "requirements", "tests", "tools"]
-    )
-    + ["fiftyone.recipes", "fiftyone.tutorials"],
+    ),
     package_dir={
```
Comment (benjaminpkane) [inline setup.py:142]: Awesome! I believe we can remove `package_dir` kwarg, as well

Comment (minhtuev) [reply setup.py]: Done!

**Review by benjaminpkane — APPROVED** [review]
LGTM 🙇

**Review by sashankaryal — APPROVED** [review]
🚀

**Pull request merged**